docker/data-science/notebooks/aips.py

```diff
@@ -13,25 +16,31 @@
 AIPS_ZK_PORT= os.getenv('AIPS_ZK_PORT') or '2181'
 AIPS_WEBSERVER_HOST = os.getenv('AIPS_WEBSERVER_HOST') or 'localhost'
 AIPS_WEBSERVER_PORT = os.getenv('AIPS_WEBSERVER_PORT') or '2345'
+ENGINE = SolrEngine()
```

Comment:
I have a feeling we need to read/write the config from/to disk. Otherwise I think each notebook will create a new aips.py since each notebook will have it's own kernel (execution environment).

If I'm right (please confirm; I'm not confident), then we should persist the config to disk to it applies across all notebooks. Would be painful to have to call "change_engine" in every single notebook.

Reply:
Just verified that Jupyter notebooks does load per notebook, so persistence will be needed- Whether it's config or in the aips.py module itself